model-optimizer/requirements_dev.txt

```diff
@@ -1,9 +1,5 @@
 coverage==4.4.2
-m2r==0.1.12
 pyenchant==1.6.11
-isort==4.3.21
-astroid==2.1.0
```

Comment:
`astroid` version was pinned for a reason - unexpected upgrade of this module can break pylint

Reply:
Can we remove the pylint and astroid from the requirements_dev.txt since it is actually not needed in the MO and keep the required version in the CI config only?

Reply:
We can, although IMO pylint is needed for MO and developer's requirements file. It is good practice to check the code before `git commit`. Pinning version in MO developer's requirements allows us to standardize pylint behavior for all MO developers.

Reply:
Technically the pylint errors are extremely rare. But I agree with you. So I have restore dependencies to the pylint in the MO requirements_dev.txt.